Week01/Problem05/mfahrbach_05.py

```diff
@@ -0,0 +1,6 @@
+
+from math import gcd
```

Comment:
Very clever, and very Pythonic...definitely more advantageous to import someone else's code (as long as you trust it!) instead of re-inventing the wheel.